Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.yml

```diff
@@ -0,0 +1,162 @@
+category: Messaging
+commonfields:
+  id: Mail Listener v2
+  version: -1
+configuration:
+- display: Mail Server Hostname / IP address
+  name: MailServerURL
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: '143'
+  display: IMAP Port
+  name: port
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: INBOX
+  display: Incoming mail folder
+  name: folder
+  required: true
+  type: 0
+- display: Fetch mails from these senders addresses only (eg. admin@demo.com,test@demo.com)
+  name: permittedFromAdd
+  required: false
+  type: 0
+- display: Fetch mails from these domains only (eg. demo.com, test.com)
+  name: permittedFromDomain
+  required: false
+  type: 0
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+- defaultvalue: '50'
+  display: The maximum number of incidents to fetch each time
+  name: limit
+  required: false
+  type: 0
+- display: Delete processed emails
+  name: delete_processed
+  required: false
+  type: 8
+- display: Include raw body in incidents
+  name: Include_raw_body
+  required: false
+  type: 8
+- display: Save the email .eml file
+  name: save_file
+  required: false
+  type: 8
+- display: Use TLS for connection (defaults to True)
+  name: TLS_connection
+  required: false
+  defaultvalue: "true"
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+description: Listens to a mailbox and enables incident triggering via e-mail.
+display: Mail Listener v2
+name: Mail Listener v2
+script:
+  commands:
+  - deprecated: false
+    description: Fetches mails according to the configuration
+    execution: false
+    name: mail-listener-list-messages
+    outputs:
+    - contextPath: MailListener.EmailPreview.Subject
```

Comment:
need to make sure we output to the context path, `Email` and not `EmailPreview`

Reply:
But the objects returned are not the same.
In list only `Subject`, `To`, `From`, `Date` and `ID` are returned.
In get email command the full email is returned.
Since they are not the same I thought is best to put them in different path in the context.

Reply:
if the same data is populated then it should be the same path, the number of fields returned does not really matter